api/v1beta1/safemode.go

```diff
@@ -23,10 +23,10 @@ type Config struct {
 // CountTooLargeConfig represents the configuration for the countTooLarge safetynet
 // +ddmark:validation:AtLeastOneOf={NamespaceThreshold,ClusterThreshold}
 type CountTooLargeConfig struct {
-	// +ddmark:validation:Minimum=0
+	// +ddmark:validation:Minimum=1
 	// +ddmark:validation:Maximum=100
 	NamespaceThreshold int `json:"namespaceThreshold,omitempty"`
```

Comment:
Can we combine a minimum of 1 and `omitempty` in this way?

Reply:
Not that I could find.

Reply:
Added a new commit which better combines the two.